### ISSUE TITLE
select pipe not found in angular 4.3.5

### ISSUE THREAD

**Issue by manzonif** [apollographql/apollo-angular#365]
Hi, trying to use select pipe into template, cause me the following error:
 Template parse errors:
The pipe 'select' could not be found.

If I declare it in my app.module.ts:

"Type SelectPipe is part of the declarations of 2 modules: ApolloModule and AppModule! Please consider moving SelectPipe to a higher module that imports ApolloModule and AppModule. You can also create a new NgModule that exports and includes SelectPipe then import that NgModule in ApolloModule and AppModule."

**Comment by kamilkisiela** [comment]
This error says that you have SelectPipe in two places, in `ApolloModule` which is a expected behavior and  in `AppModule` and that's weird. 

Let me explain it to you, using a code sample:

```ts
@NgModule({
  declarations: [ SelectPipe ],
  imports: [ ApolloModule ]
})
class AppModule {}
```

Is this what you have?

**Comment by manzonif** [comment]
without SelectPipe in declarations:

Template parse errors:
The pipe 'select' could not be found.****

**Comment by JCBsystem** [comment]
any new on this ?
have the same problem

**Comment by kamilkisiela** [comment]
Fixed in v1.0
